### PR TITLE
{vis}[GCCcore/11.3.0] pygraphviz v1.10 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

Changelog:
* move down to `GCCcore` as graphviz moved down as well
* converted to PythonBundle

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.10-GCCcore-11.3.0.eb
Comment (casparvl) [inline easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.10-GCCcore-11.3.0.eb:1]: Can you comment (just here in the PR, it doesn't need to end up in the EasyConfig) _why_ you changed this to PythonBundle? I don't particularly mind - if it works it works - but it might be good to log it for posterity :)

Comment (casparvl) [reply easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.10-GCCcore-11.3.0.eb:1]: You know what, I'll not block the merge over this. Still, would be nice to add it as a comment after the merge.

**Review by casparvl — APPROVED** [review]

**Comment by casparvl** [comment]
Going in, thanks @lexming!

File: easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.10-GCCcore-11.3.0.eb
Comment (lexming) [reply easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.10-GCCcore-11.3.0.eb:1]: Right now is mostly a matter of preference, basically trying to consolidate around a single easyblock for all installations of Python packages. There is an open issue about this #15639 . But it is not mandatory (yet).

**Pull request merged**
